doc/cephfs/dirfrags.rst

```diff
@@ -58,14 +58,15 @@ ENOSPC errors if they try to create files in the fragment.  On
 a properly configured system, this limit should never be reached on
 ordinary directories, as they will have split long before.  By default,
 this is set to 10 times the split size, giving a dirfrag size limit of
-100000.  Increasing this limit may lead to oversized directory fragment
-objects in the metadata pool, which the OSDs may not be able to handle.
+100000 omap entries.  Increasing this limit may lead to oversized directory
```

Comment:
This should also be changed to 'directory entries' or 'entries'? @kotreshhr

Reply:
Yes